lib/authz/permissions.go

```diff
@@ -14,11 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package auth
+package authz
```

Comment:
WDYT about `package authorizer`? Fits "most" of what's defined here, and seems a bit smaller in scope than `authz`.